kolibri/core/discovery/utils/filesystem/posix.py

```diff
@@ -30,20 +35,25 @@ def get_drive_list():
     Disk size/usage comes from shutil.disk_usage or os.statvfs, and name/type info from dbus (Linux) or diskutil (OSX).
     """
 
-    drivelist = subprocess.Popen('mount', shell=True, stdout=subprocess.PIPE)
-    drivelisto, err = drivelist.communicate()
-
-    drives = []
-
     if sys.platform == "darwin":
         MOUNT_PARSER = OSX_MOUNT_PARSER
     else:
         MOUNT_PARSER = LINUX_MOUNT_PARSER
 
+    try:
+        drivelist = subprocess.Popen('mount', shell=True, stdout=subprocess.PIPE)
+        drivelisto, err = drivelist.communicate()
+    except OSError:  # couldn't run `mount`, let's try reading the /etc/mounts listing directly
+        with open("/proc/mounts") as f:
+            drivelisto = f.read()
```

Comment:
It doesn't exist on Windows, and I think you mentioned that this part was scoped out for now? Anyways, it's nice to have these things somehow contained.

A patter could for instance be:

```
    if sys.platform == "darwin":
        parser = darwin_parser
     elif sys.platform == "posix":
         parser = linux_parser
     else:
         parser = windows_parser

# ...

def windows_parser():
     raise NotImplemented("Windows isn't supported")
```